common/config.go

```diff
@@ -65,21 +65,17 @@ func MainnetConfig() *Config {
 		EthEndpoint: "", // No mainnet default (permissionless URLs are not reliable)
 		MoneroNodes: []*MoneroNode{
 			{
-				Host: "node.sethforprivacy.com",
```

Comment:
The sethforprivacy nodes, both stagenet and mainnet, have been offline for long enough that I don't think they are comming back up.